### DISCUSSION TITLE
model_json_schema(schema_generator=xxxx) examples wanted!

### DISCUSSION THREAD

**Discussion by tomvonheill** [pydantic/pydantic#7256] [General]
Anyone have an example of how they used schema_generator to change the out put of model_json_schema? Currently I'm outputting the dictionary and replacing keys/values but would love to use the schema_generator value instead.
<img width="837" alt="image" src="https://github.com/pydantic/pydantic/assets/10145413/c9bca08d-2ad5-4e95-94e6-058ae6338ce0">

**Comment by harunyasar** [comment]
```python
from pydantic import BaseModel
from pydantic.json_schema import GenerateJsonSchema

class MyCustomJsonSchema(GenerateJsonSchema):
    ...  # do your own implementation by following https://docs.pydantic.dev/latest/api/json_schema/

class MyModel(BaseModel):
    id: int
    name: str

schema = MyModel.model_json_schema(schema_generator=MyCustomJsonSchema)
```

Is that what you are looking for?
